core/src/main/java/tech/pegasys/eth2signer/core/Runner.java

```diff
@@ -278,6 +279,9 @@ private HttpServer createServerAndWait(
         new HttpServerOptions()
             .setPort(config.getHttpListenPort())
             .setHost(config.getHttpListenHost())
+            .setTcpKeepAlive(false)
+            .setIdleTimeout(5)
```

Comment:
Default seems a bit low. I think 30s at least would be a better default

Reply:
made this configurable